website/content/docs/agent/telemetry.mdx

```diff
@@ -541,6 +541,7 @@ These metrics are used to monitor the health of the Consul servers.
 | `consul.grpc.server.streams`                        | Measures the number of active gRPC streams handled by the server. Includes a `server_type` label indicating either the `internal` or `external` gRPC server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | streams                           | gauge   |
 | `consul.xds.server.streams`                         | Measures the number of active xDS streams handled by the server split by protocol version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | streams                           | gauge   |
+| `consul.xds.server.streamsUnauthenticated`          | Measures the number of active xDS streams handled by the server that are unauthenticated because ACLs are not enabled or ACL tokens were missing.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | streams                           | gauge   |
```

Comment:
What does this gauge look like if ACLs are enabled. Will we have any (long-lived) xDS streams that are unauthenticated? Or would this basically be a gauge of the attempts to open an xDS stream that are unauthenticated?

Reply:
Whether ACLs are on or off this gauge will track streams that do not have tokens associated with them. I'm not entirely sure if any unauthenticated streams will be long-lived, unless the anonymous token is privileged enough to continue operating.